### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by sophiewigmore**
## Summary

#4 

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by sophiewigmore** [comment]
Open to feedback around the wording, especially in the integration section.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Paketo Buildpack for .NET Core ASP.NET Runtime
```
Comment (fg-j) [inline README.md:1]:
```suggestion
# Paketo Buildpack for ASP.NET Core Runtim
```
For clarity, let's keep the naming consistent with what's baked into the `buildpack.toml`

Comment (fg-j) [reply README.md:1]: This applies wherever we refer to the name of the buildpack or the name of the dependency it installs, "ASP.NET Core Runtime".

**Review by fg-j — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sophiewigmore) [reply README.md]: Fixed everywhere!

**Review by fg-j — APPROVED** [review]

**Pull request merged**